pom.xml

```diff
@@ -0,0 +1,602 @@
+        </developer>
+    </developers>
+    <scm>
+        <developerConnection>scm:git:https://github.com/lafaspot/ja3_4java.git</developerConnection>
+        <connection>scm:git:https://github.com/lafaspot/ja3_4java.git</connection>
+        <url>https://github.com/lafaspot/ja3_4java.git</url>
+        <tag>ja3_4java-.1</tag>
+    </scm>
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+    <properties>
+        <project-name>ja3_4java</project-name>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>${project.build.sourceEncoding}</project.reporting.outputEncoding>
+        <excludedJNILibraries />
+        <excludedWarJNILibraries />
+        <jdk.version>1.8</jdk.version>
+        <maven-findbugs-plugin.version>3.0.1</maven-findbugs-plugin.version>
+        <maven-surefire-plugin.version>2.17</maven-surefire-plugin.version>
+        <maven-pmd-plugin.version>3.4</maven-pmd-plugin.version>
+        <maven-checkstyle-plugin.version>2.13</maven-checkstyle-plugin.version>
+        <jersey.version>2.18</jersey.version>
+        <jackson.version>2.5.3</jackson.version>
+        <jacoco-maven-plugin.version>0.7.5.201505241946</jacoco-maven-plugin.version>
+        <main.basedir>${project.basedir}</main.basedir>
+        <testTimeLimitInSeconds>5</testTimeLimitInSeconds>
+        <checkstyle.maxAllowedViolations>25</checkstyle.maxAllowedViolations>
+    </properties>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>6.8.8</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <artifactId>junit</artifactId>
+                    <groupId>junit</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.10.19</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>1.10.19</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
```

Comment:
wouldn't this get packaged if scope is non test?

Reply:
removed